Eatery/Onboarding/Controllers/OnboardingViewController.swift

```diff
@@ -14,24 +14,21 @@ protocol OnboardingViewControllerDelegate {
 
 class OnboardingViewController: UIViewController {
 
-    let stackView = UIStackView()
-    let subtitleLabel = UILabel()
-    
+    private let stackView = UIStackView()
     private let titleLabel = UILabel()
-    private var imageView: UIImageView!
-    private let nextButton = UIButton()
+    private let subtitleLabel = UILabel()
 
     private var onboardingTitle: String!
     private var onboardingSubtitle: String!
```

Comment:
Like my first comment, remove the `!` and move lines 30 and 31 before the super call.